app/http.php

```diff
@@ -1491,25 +1491,30 @@ function (string $runtimeId, ?string $payload, string $path, string $method, mix
      */
     $allowList = empty(Http::getEnv('OPR_EXECUTOR_RUNTIMES')) ? [] : \explode(',', Http::getEnv('OPR_EXECUTOR_RUNTIMES'));
 
-    $runtimeVersions = \explode(',', Http::getEnv('OPR_EXECUTOR_RUNTIME_VERSIONS', 'v4') ?? 'v4');
-    foreach ($runtimeVersions as $runtimeVersion) {
-        Console::success("Pulling $runtimeVersion images...");
-        $runtimes = new Runtimes($runtimeVersion); // TODO: @Meldiron Make part of open runtimes
-        $runtimes = $runtimes->getAll(true, $allowList);
-        $callables = [];
-        foreach ($runtimes as $runtime) {
-            $callables[] = function () use ($runtime, $orchestration) {
-                Console::log('Warming up ' . $runtime['name'] . ' ' . $runtime['version'] . ' environment...');
-                $response = $orchestration->pull($runtime['image']);
-                if ($response) {
-                    Console::info("Successfully Warmed up {$runtime['name']} {$runtime['version']}!");
-                } else {
-                    Console::warning("Failed to Warmup {$runtime['name']} {$runtime['version']}!");
-                }
-            };
-        }
+    if (Http::getEnv('OPR_EXECUTOR_IMAGE_PULL', 'enabled') === 'disabled') {
+        // Useful to prevent auto-pulling from remote when testing local images
+        Console::info("Skipping image pulling");
+    } else {
```

Comment:
```suggestion
    if (Http::getEnv('OPR_EXECUTOR_IMAGE_PULL', 'enabled') === 'disabled') {
        // Useful to prevent auto-pulling from remote when testing local images
        Console::info("Skipping image pulling");
        return;
    } 
```

Can we do something like this to remove the else block and make a bit less indentation to make it readable?

Reply:
Sadly not in this scenario, code continues to start http server.

I like the idea, and it is possible to allow this, but that would make PR diff even bigger and more complex. I would like to avoid that as I want this PR merged quickly.